.github/CHANGELOG.md

```diff
@@ -2,6 +2,28 @@
 
 <h3>New features since last release</h3>
 
+* The new ``qml.apply_op`` function can be used to add operations that might have
+  already been instantiated elsewhere to the QNode:
+  [(#1433)](https://github.com/PennyLaneAI/pennylane/pull/1433)
+
+  ```python
+  op = qml.RX(0.4, wires=0)
+  dev = qml.device("default.qubit", wires=2)
+
+  @qml.qnode(dev)
+  def circuit(x):
+      qml.RY(x, wires=0)
+      qml.apply_op(op)
+      return qml.expval(qml.PauliZ(0))
+  ```
+
+  ```pycon
+  >>> print(qml.draw(circuit)(0.6))
+  0: ──RY(0.6)──RX(0.4)──┤ ⟨Z⟩
+  ```
+
+  Previously instantiated measurements can also be applied to QNodes.
```

Comment:
This is one of the reasons I prefer `qml.apply` rather than `qml.apply_op` :(

Reply:
Yeah I strongly prefer just `qml.apply`. Plus `apply_op(op)` looks kind of redundant. How much overhead would it be to rename the other `apply` (and is there another name that makes sense?).

Alternatively, what about `qml.do` or  `qml.implement`?

Reply:
I think it would be relatively easy to replace the existing `qml.apply`, since I don't recall us ever using it publicly in demos/examples?

Reply:
But it would still be a breaking change

Reply:
Also in favour. I recently came across a decorator that allows us to define functions with the same name but different signatures, but I forgot how it was called (`@distribue` or so?).

In principle we could use something like this in future?